Lib9c.DevExtensions/Action/Craft/UnlockCraftAction.cs

```diff
@@ -0,0 +1,78 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using Bencodex.Types;
+using Libplanet;
+using Libplanet.Action;
+using Nekoyume;
+using Nekoyume.Action;
+using Nekoyume.Exceptions;
+using Nekoyume.Model;
+using Nekoyume.Model.State;
+using Nekoyume.TableData;
+using static Lib9c.SerializeKeys;
+
+namespace Lib9c.DevExtensions.Action.Craft
+{
+    [Serializable]
+    [ActionType("unlock_craft_action")]
+    public class UnlockCraftAction : GameAction
+    {
+        public Address AvatarAddress { get; set; }
+        public ActionTypeAttribute ActionType { get; set; }
```

Comment:
Can we change the type of `ActionType` to `string`? Because this action needs only the `ActionTypeAttribute.TypeIdentifier`.🙂